src/components/layout/Header.tsx

```
@@ -43,25 +51,65 @@ export function SearchHeader() {
+                  // Not logged in
+                  <Button
+                    variant="ghost"
+                    size="sm"
+                    onClick={() => router.push('/login')}
+                    className="flex items-center space-x-1"
+                  >
+                    <User className="w-4 h-4" />
+                    <span>Login</span>
+                  </Button>
+                )}
+              </>
```

Biome (1.9.4)

[error] Avoid using unnecessary Fragment.

A fragment is redundant if it contains only one child, or if it is the child of a html element, and is not a keyed fragment.
Unsafe fix: Remove the Fragment


(lint/complexity/noUselessFragments)